app/controllers/projects_controller.rb

```diff
@@ -0,0 +1,30 @@
+class ProjectsController < ApplicationController
+  def index
+    @projects = Project.all
+  end
+
+  def show
+    @project = Project.find(params[:id])
+  end
+
+  def new
+    @project = Project.new
+  end
+
+  def create
+    @hackday = Hackday.find(project_params[:hackday_id])
```

Comment:
params come in as strings. you should do to_i on them otherwise the mysql query is a lot more inefficient.